src/System.CommandLine/Binding/ModelBinder.cs

```diff
@@ -250,7 +251,7 @@ public void UpdateInstance<T>(T instance, BindingContext bindingContext)
                 return valueSource;
             }
 
-            if (bindingContext.TryGetValueSource(valueDescriptor, out valueSource))
+            if (bindingContext.TryGetValueSource(valueDescriptor, out valueSource!))
```

Comment:
Consider changing the `var` above to be an explicit type with nullable annotation instead of using a suppression here.
In 16.6, `var` will automatically be nullable and this problem should go away.

I'm actually surprised there is no warning on `TryGetValue` above... If the `var` is inferred to be not nullable, then there should be a warning for one of the two return cases.